### PR TITLE
Use for loop to create custom amount of rounds and shots.

### PR DESCRIPTION
**Pull request by VeronikaHar**
Rounds and shots will be passed in props or redux store later.

### CODE REVIEW

File: src/components/Scoreboard.tsx [hunk 1]
```diff
@@ -162,10 +164,10 @@ const styles = StyleSheet.create({
 		justifyContent: "space-between",
 		width: wp("100%"),
 		height: hp("33%"),
-		backgroundColor: "#98ddff",
+		backgroundColor: "#B3A210",
 	},
 	summary: {
-		backgroundColor: "#5cd5ff",
+		backgroundColor: "#998B0E",
 		width: wp("100%"),
```
Comment (pncifra) [inline src/components/Scoreboard.tsx]: this looks good.

File: src/components/Scoreboard.tsx [hunk 2]
```diff
@@ -155,9 +153,9 @@ const styles = StyleSheet.create({
 		bottom: 1,
 		flexDirection: "row",
 		flexWrap: "wrap",
-		justifyContent: "space-between",
+		justifyContent: "space-evenly",
 		width: wp("100%"),
-		height: hp("27.5%"),
+		height: hp("25.8%"),
 		backgroundColor: "#34558b",
```
Comment (pncifra) [inline src/components/Scoreboard.tsx]: looks good, a nit: i'd add some comments in the code prior to methods , etc just good practice. At the very least top level comments describing the component

File: src/components/Scoreboard.tsx [hunk 3]
```diff
@@ -70,7 +70,7 @@ export class ScoreboardScreen extends React.Component {
 					</TouchableHighlight>
 					<TouchableHighlight
 						style={[styles.key, { backgroundColor: "#34559d" }]}
-						onPress={() => { this.state.scores.pop(), console.log(this.state.scores) }}>
+						onPress={() => { this.setState({ scores: [...this.state.scores.slice(0, -1)] }) }}>
 						<Text style={styles.keytext}>Del</Text>
```
Comment (pncifra) [inline src/components/Scoreboard.tsx]: yups

File: src/components/Scoreboard.tsx [hunk 4]
```diff
@@ -13,39 +13,37 @@ export class ScoreboardScreen extends React.Component {
...
+					value={"Round " + i + ":  " + this.state.scores.slice(this.state.shots * (i - 1)).toString().split(",").join("  ")}
+				/>)
+		}
+		return rounds;
+	}
+
+	render() {
+		return (
+			<View style={{ flex: 1 }}>
+				{this.createRounds()}
```
Comment (pncifra) [inline src/components/Scoreboard.tsx]: yups

File: src/components/Scoreboard.tsx (hunk 2 above)
Comment (VeronikaHar) [reply src/components/Scoreboard.tsx]: Done!

**Pull request merged**